libs/payments/ui/src/lib/server/subscription-title.tsx

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from 'next/image';
+import { CartState } from '@fxa/shared/db/mysql/account';
+import { LocalizerRsc } from '@fxa/shared/l10n/server';
+import checkLogo from '../../images/check.svg';
+
+interface SubscriptionTitleProps {
+  l10n: LocalizerRsc;
+  pageType: CartState;
+}
+
+const getComponentTitle = (pageType: CartState) => {
+  switch (pageType) {
+    case CartState.FAIL:
+      return {
+        title: 'Error confirming subscription…',
+        titleFtl: 'next-subscription-error-title',
+      };
+    case CartState.PROCESSING:
+      return {
+        title: 'Confirming subscription…',
+        titleFtl: 'next-subscription-processing-title',
+      };
+    case CartState.SUCCESS:
+      return {
+        title: 'Subscription confirmation',
+        titleFtl: 'next-subscription-success-title',
+      };
+    case CartState.START:
+    default:
```

Comment:
Should the `default` case throw an error in the off-chance that we have an invalid `pageType`?

Reply:
After a discussion with Reino - 
-  I am going to update the variable `pageType` to `cartState` (to avoid confusion as that is what it actually is). 
- As the value should always be one of the cartStates, there shouldn't be an invalid state, but I'll add a console.error for now

Thanks Peter!